File: 2_Kernel/JavaKernel/src/main/java/com/seagosoft/tws/data/UpdateMktDepthData.java
package com.seagosoft.tws.data;

import com.ib.client.Decimal;

public class UpdateMktDepthData {

    public int tickerId;
    public int position;
    public int operation;
    public int side;
    public double price;
    public Decimal size;

    public UpdateMktDepthData(int tickerId, int position, int operation, int side, double price, Decimal size) {
        this.tickerId = tickerId;
        this.position = position;
        this.operation = operation;
        this.side = side;
        this.price = price;
        this.size = size;
    }
}

File: 3_OrderSystem/KotlinEWrapperBackend/src/main/kotlin/com/seagosoft/tws/wrapper/structs/UpdateMktDepthData.kt
package com.seagosoft.tws.wrapper.structs

import com.ib.client.Decimal


class UpdateMktDepthData(
    var tickerId: Int,
    var position: Int,
    var operation: Int,
    var side: Int,
    var price: Double,
    var size: Decimal
)